+fishshell.com

jhheider opened the pull request.

jhheider commented:
[ready to go](https://www.youtube.com/watch?v=ph4rpxsu6SU), @Xercesblu3 (definitely use squash)

mxcl commented:
Please add a comment explaining why the diff is necessary.

We need a bot that whatever @jhheider does it demands he adds comments.

jhheider committed:
+fishshell.com


-DCURSES_INCLUDE_PATH


do that better


it thinks it wants pkg-config, i guess.


whyyyyyyyyyyyyyyy


keep make


needs  these at runtime


annotate diff

jhheider commented:
> Please add a comment explaining why the diff is necessary.
> 

Done.

> We need a bot that whatever @jhheider does it demands he adds comments.

No way. Certainly it's no more than half of my commits. ;)
